### ISSUE TITLE
Addon broken on alt characters

### ISSUE THREAD

**Issue by fdeters** [Justw8/TeleportMenu#42]
**Version:** 8.3

The addon works fine on my main character, but doesn't work on my alts. (Not sure why there would be a distinction, other than that my main character is probably the one who first used the addon).

Here are the problems I've noticed on the characters that have issues:

- Addon does not appear in the Options menu.
- The Escape menu shows the "random hearthstone" button, but not the regular hearthstone.
  - The random hearthstone button doesn't do anything when you click on it
  - (I turned on the random hearthstone setting on my main character, so it's not a problem that I'm seeing it)
- The console command menu works, but sometimes `/tpm list` must be run more than once to see output from it.

Very spooky! Like I said, things are working on my main character. No idea why the others would have an issue!

**Comment by fdeters** [comment]
Sure thing! This is the error + stack trace that happens when I log in on my alts. No errors on my main.

```lua
1x bad argument #1 to '?' (Usage: local startTimeSeconds, durationSeconds, enableCooldownTimer = C_Item.GetItemCooldown(itemInfo))
[string "=[C]"]: in function `GetItemCooldown'
[string "@TeleportMenu/TeleportMenu.lua"]:499: in function `CheckCooldown'
[string "@TeleportMenu/TeleportMenu.lua"]:646: in function <TeleportMenu/TeleportMenu.lua:555>
[string "@TeleportMenu/TeleportMenu.lua"]:787: in function `Setup'
[string "@TeleportMenu/TeleportMenu.lua"]:766: in function <TeleportMenu/TeleportMenu.lua:764>
[string "=[C]"]: in function `xpcall'
[string "@Blizzard_ObjectAPI/Mainline/ContinuableContainer.lua"]:83: in function <...Blizzard_ObjectAPI/Mainline/ContinuableContainer.lua:75>
[string "=(tail call)"]: ?
[string "@TeleportMenu/TeleportMenu.lua"]:773: in function <TeleportMenu/TeleportMenu.lua:747>
[string "@TeleportMenu/TeleportMenu.lua"]:797: in function <TeleportMenu/TeleportMenu.lua:791>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: local startTimeSeconds, durationSeconds, enableCooldownTimer = C_Item.GetItemCooldown(itemInfo))"
```

**Comment by Justw8** [comment]
I added a prevention of this error, but please let me know if something weird is still happening.

**Comment by Justw8** [comment]
(closing for now, open again if things continue)

**Comment by fdeters** [comment]
Confirmed that this is fixed on 9.0. It works on my alts again! Thanks for your work.
